cmd/ww/server.go

```diff
@@ -287,6 +293,8 @@ func server(args ...string) {
 	stunservers := set.String("stun", "stun:relay.webwormhole.io", "list of STUN server addresses to tell clients to use")
 	set.StringVar(&turnServer, "turn", "", "TURN server to use for relaying")
 	set.StringVar(&turnSecret, "turn-secret", "", "secret for HMAC-based authentication in TURN server")
+	set.StringVar(&certFlag, "cert", "", "Certificate for HTTPS (leave empty to use letsencrypt)")
+	set.StringVar(&certKeyFlag, "key", "", "Certificate key")
```

Comment:
local variables should be fine here:
```
   cert := set.String("cert", ....
   key := set.String("key", ....
```

(we'll have to use `*cert` and `*key` below instead of `cert` and `key`)

Reply:
Absolutely correct, removed the global variables and using flags instead.